### ISSUE TITLE
Feature: Change channel name of #tools-and-devops

### ISSUE THREAD

**Issue by Auto-5** [python-discord/meta#105]
# What?

Change the channel name of `#tools-and-devops` to `#hosts-and-devops`.

# Justification

What can you hear when you think the word `tools and devops`?

The word definitions to the average coding user:
- `tools` is something to help
- `devops` is an acronym for development and operations

However in the context of  senior programming they mean:
- `tools` is something to help your programming
- `devops` is the things developers need to do

The channel is for conversations about:
- version control
- dependency management
- packaging
- virtual environment
- continuous integration
- deployment
- monitoring

From the word `tools` you would never think of the above until the channel description told you since tools is too general in programming. 

The word `DevOps` includes all of them but many users are not familiar with formal programming which is expected as discord itself is informal and this means users can only think it means the things developers must do but the users will not actually know what those things are.

Because of the wording of the channel, 99% of the time users will have to click the channel and read the channel description to know what the channel is about because the name is not clear enough.

The solution is to replace `tools` with `hosts`. 

This is because `tools` is a useless word as it cant represent the channel and does not help the other formal term `DevOps` but only helps make the channel name look more wordy without actually helping to explain it. 

However the word `hosts` is defined to the average user as:
- `host` the thing to be run on

This word is very well known in the server, even to a newbie! Intermediate users will immediately wonder how or what hosts are able to handle `version control` or successfully install `dependencies` for their projects. They will continue to think things like it's a `virtual environment`, how to `deploy` their projects and if their host provides `monitoring` services. This all connects with devops even if the user doesn't actually know it. 

In conclusion I think putting `hosts` in the channel name is more self explanatory than `tools`. `DevOps` is limited to a few users because most people dont know what topics that relates to. This is why `hosts and devops` is a better combination.

Additional facts:
- The channel only gets `~10 messages` per day, a conversation never happens concurrently. This is because users dont want to investigate channel descriptions to find the right channel to use.
- Users keep talking about hosts in many channels but never in this one. You can search for `host` as evidence.
- Since `hosts` and `devops` topics are very similar users will give attention to extra topics on the devops only side of the channel and more people will talk about it.

After this implementation all of these problems will be resolved.

# Implementation

1. The community will reply to this Github issue with what they think about it.
2. The staff team of the Python discord server will have a meeting to discuss about this change.
3. One of the admins from the Python discord server will change the channel name.

**Comment by MarkKoz** [comment]
This channel surprisingly doesn't go too off-topic, in my experiences. Perhaps the most common misunderstanding I see is asking questions about using some library in there. I agree "tools" is vague, but it's sort of the point to keep it general. Unfortunately, there isn't something both accurate and succinct.

Sorry, but I am not sold at all on your suggestion of "hosts". To me, it heavily implies something related to deployment (e.g. where or how to host a website). However, the channel is not only about deployment and deployment tools. I'm going to use version control as an example here. What would git have to do with hosts? It's a development tool, not a deployment tool. One could argue "devops" encompasses that, but as you wrote, devops is not a term most people understand. Furthermore, by that logic, "devops" would already encompass deployment too.

> what hosts are able to handle version control

This doesn't make sense. As explained above, hosts don't deal with version control; it's a tool for developers. Unless you're using host in the sense of the developer's machine, but that would be an odd usage of it in a general context.

> Because of the wording of the channel, 99% of the time users will have to click the channel and read the channel description to know what the channel is about because the name is not clear enough.

Are you writing this from your own experiences trying to understand the channel's name, or are you making inferences on how others would understand it? Where is this "99%" statistic coming from?

> From the word tools you would never think of the above until the channel description told you since tools is too general in programming.

Well, I certainly would for some of that stuff (the rest would be covered by "devops"). What evidence do you have to support that other's wouldn't? As mentioned earlier, I do think people conflate "tool" and "library". Maybe a clearer word exists, but I don't think "hosts" is the solution to that problem.

> "tools is something to help" vs "tools is something to help your programming"

It's a server for developers and about development. I don't see how one would reasonably understand "tools" as something other than ones related to development, so I don't understand what kind of distinction you're trying to make.

**Comment by jb3** [comment]
We discussed this within a staff meeting and out of the names proposed by this issue and other ones we came up with none were more suitable than the current name.

We don't want to bikeshed names too much and this covers the vast majority of topics within the channel, worst case scenario someone asks a topic outside #tools-and-devops and is directed there.

I'll be closing this issue now since no action is required.
